src/components/InputTextbox/index.jsx

```diff
@@ -0,0 +1,22 @@
+import { useState } from "react";
+
+export default function index(props) {
+  const [email, setEmail] = useState('');
+
+  const handleChange = e => {
+    setEmail(e.target.value);
+  };
+
+  return (
+    <div>
+      <input
+        type='text'
+        onChange={handleChange}
```

Comment:
No state should be handled in this component. This component is and only is a representation and styling of an input field which should pass the onChange back to the parent through a callback function as we had discussed previously.

Reply:
Updated as requested.